docs/sphinx/user_guide/config_options.rst

```diff
@@ -137,16 +137,23 @@ and their default settings:
       Variable                 Default
       ======================   ==========================================
       ENABLE_CLANG_CUDA        Off (when on, ENABLE_CUDA must also be on)
-      ENABLE_CUB               On (when CUDA enabled)
+      ENABLE_EXTERNAL_CUB      Off (when CUDA enabled)
```

Comment:
Is ENABLE_EXTERNAL_ROCPRIM also dumented in this file?

Reply:
I'm not sure, but its behavior is not changed by this PR.

Reply:
Apparently it is not documented. Would you please add documentation for that option in the table? Thank you.

Reply:
added